haystack/management/commands/build_solr_schema.py

```diff
@@ -94,16 +94,16 @@ def handle(self, **options):
                 raise CommandError('Could not write {}: {}'.format(solrconfig_path, exc))
 
         if reload_core:
-            core = settings.HAYSTACK_CONNECTIONS['solr']['URL'].rsplit('/', 1)[-1]
+            core = settings.HAYSTACK_CONNECTIONS[using]['URL'].rsplit('/', 1)[-1]
 
-            if 'ADMIN_URL' not in settings.HAYSTACK_CONNECTIONS['solr']:
                 raise ImproperlyConfigured("'ADMIN_URL' must be specified in the HAYSTACK_CONNECTIONS settings for the backend")
-            if 'URL' not in settings.HAYSTACK_CONNECTIONS['solr']:
                 raise ImproperlyConfigured("'URL' to the core must be specified in the HAYSTACK_CONNECTIONS settings for the backend")
+            if 'ADMIN_URL' not in settings.HAYSTACK_CONNECTIONS[using]:
```

Comment:
I don't think I can commit on this branch, so I wanted to point out the minor syntax error here!